client-js/app/elements/labrad-plot.html

```diff
@@ -41,12 +41,55 @@
       <paper-icon-button icon="tab-unselected" id="rect" on-click="mouseModeZoomRect" title="drag to zoom"></paper-icon-button>
       <paper-icon-button icon="open-with" id="pan" on-click="mouseModePan" title="drag to pan"></paper-icon-button>
       <paper-icon-button icon="home" id="home" on-click="resetZoom" title="zoom out"></paper-icon-button>
-
+      <paper-icon-button icon="list" id="select" on-click="selectTraces" title="select trace"></paper-icon-button>
       <div id="modes2d">
         <paper-icon-button icon="av:stop" id="dots" on-click="drawMode2DDots" title="draw with dots"></paper-icon-button>
         <paper-icon-button icon="dashboard" id="rectfill" on-click="drawMode2DRectfill" title="draw with rect fill"></paper-icon-button>
         <paper-icon-button icon="view-list" id="vargrid" on-click="drawMode2DVargrid" title="draw with vargrid"></paper-icon-button>
       </div>
     </div>
+    
+    <paper-dialog id="traceSelector">
+      <h1>Select Traces</h1>
+      <paper-dialog-scrollable>
+        <form is="iron-form" id="selectForm">
+          <template is="dom-if" if="{{oneIndep}}">
+            <template is="dom-repeat" items="{{deps}}">
+              <div>
+                <!-- The 's' in front on {{index}} is a hack around a stupid javascript thing 
+                where value="0" makes the checkbox not exist, value="s0" counts as a label :/ -->
+                <paper-checkbox checked name="traces" value="s{{index}}">{{index}}</paper-checkbox>
+                <span> - </span><span>{{item.label}}</span>
+                <span> - </span>
+                <span>{{item.legend}}</span> [<span>{{item.unit}}</span>]
+              </div>
+            </template>
+            <div id="selectButtons">
+              <paper-button on-click="selectAll">select all</paper-button>
+              <paper-button on-click="deselectAll">select none</paper-button>
+            </div>
+          </template>
+          <template is="dom-if" if="{{!oneIndep}}">
+            <!-- The 's' in front on {{index}} is a hack around a stupid javascript thing 
+            where value="0" makes the checkbox not exist, value="s0" counts as a label :/ -->
+            <paper-radio-group selected="s0">
+              <template is="dom-repeat" items="{{deps}}">
+                <paper-radio-button name="s{{index}}">
+                  <span>{{item.label}}</span>
+                  <span> - </span>
+                  <span>{{item.legend}}</span> [<span>{{item.unit}}</span>]
```

Comment:
The data vault formats these like `label (legend) [unit]`. I think we should follow that here as well.